src/python/src/setup.py

```diff
@@ -98,15 +104,23 @@
     'grpc.framework': 'grpc/framework',
 }
 
+_INSTALL_REQUIRES = (
+    'enum34==1.0.4',
+    'futures==2.2.0',
+    'protobuf==3.0.0a3'
+)
+
 setuptools.setup(
     name='grpcio',
     version='0.10.0a0',
     ext_modules=_EXTENSION_MODULES,
     packages=list(_PACKAGES),
     package_dir=_PACKAGE_DIRECTORIES,
-    install_requires=[
-        'enum34==1.0.4',
-        'futures==2.2.0',
-        'protobuf==3.0.0a3'
-    ]
+    install_requires=_INSTALL_REQUIRES,
+    setup_requires=(
+        'sphinx>=1.3',
+    ) + _INSTALL_REQUIRES,
```

Comment:
Let's not do this algebra inside the setuptools.setup function call. How about

_SETUP_REQUIRES = tuple(['sphinx>=1.3'] + list(_INSTALL_REQURES))

at top-level and

setup_requires=list(_SETUP_REQUIRES),

here?